CMakeLists.txt

```diff
@@ -81,7 +81,7 @@ set(OT_MBEDTLS ${OT_EXTERNAL_MBEDTLS})
 
 # RCP mbedtls library
 if(NOT OT_MBEDTLS_RCP)
-    set(OT_MBEDTLS_RCP "openthread-efr32-rcp-mbedtls" CACHE STRING "default efr32 RCP mbedtls library" FORCE)
+    set(OT_MBEDTLS_RCP "openthread-efr32-rcp-uart-mbedtls" CACHE STRING "default efr32 RCP mbedtls library" FORCE)
```

Comment:
Do we need a `openthread-efr32-rcp-spi-mbedtls` library or should this remain as `openthread-efr32-rcp-mbedtls` and its used for both spi and uart?

Reply:
See comment above.  Don't we need `openthread-efr32-rcp-spi-mbedtls`?

Reply:
That is default config set to uart one. We do set correct libs in build script.
We can do something like

if(NOT OT_MBEDTLS_RCP)
      if(OT_NCP_SPI) 
          set spi
      else 
         set uart one.
     endif
endif

Reply:
I see.  Although it may not be needed I think it would more clear to add the logic you suggested.

Reply:
Sounds good 👍, I just pushed the changes.